### DISCUSSION TITLE
how to change ae_title of dcm4chee ?

### DISCUSSION THREAD

**Discussion by hungndqps** [dcm4che/dcm4chee-arc-light#3778] [Q&A] [answered]
Hi Guys, I'm a newbie. we have a lab env of MWL Server using DCM4CHEE running on docker (minimal setup).

Its default name is DCM4CHEE which we have to register on modality as MWL SCP; we need to change that name to another one but still unable to find a way. Could you guys please give a hint or instruction that help us with this.

Thank you so much.

**Comment by vrindanayak** [comment, marked as answer]
See [AE Title(s) of the Archive](https://github.com/dcm4che/dcm4chee-arc-light/wiki/AE-Title%28s%29-of-the-Archive)

**Comment by Mimiz06** [comment]
thanks a lot, but i'm stayin' stuck.

**Comment by FullMetalEd** [comment]
Hey mate, 

You can change the AE title by editing the device; configuration > edit device > child objects > Network AE > DCM4CHEE > Attributes > AE title
![image](https://user-images.githubusercontent.com/42138052/229259446-6fd8677b-42d7-4a2a-a568-742ca89a94cb.png)


Hope that helps.

**Reply by Mimiz06** [reply]
thank you ! yes ! it helps !!!!
